tests/app/shared/issue-tables/issue-tables.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { IssueTablesComponent } from '../../../../src/app/shared/issue-tables/issue-tables.component';
+
+describe('IssueTablesComponent', () => {
+    let issueTablesComponent: IssueTablesComponent;
+
+    describe('fitTitleText()', () => {
+        beforeEach(() => {
+        issueTablesComponent = new IssueTablesComponent(null, null, null, null, null, null, null);
+        });
+
+        it('should format the title text to account for words longer than max word length', () => {
+            // title consisting of 2 words of 50 characters
```

Comment:
Not sure but I think it'll be good to add a test case for 1 word as well!

Reply:
Sure, I'll do so